templates/reputation/contracts/ReputationTemplate.sol

```diff
@@ -0,0 +1,97 @@
+pragma solidity 0.4.24;
+
+import "@aragon/templates-shared/contracts/BaseTemplate.sol";
+
+
+contract ReputationTemplate is BaseTemplate {
+    string private constant ERROR_MISSING_TOKEN_CACHE = "REPUTATION_MISSING_TOKEN_CACHE";
+    string private constant ERROR_INVALID_HOLDERS_STAKES_LEN = "REPUTATION_INVALID_HOLDERS_STAKES_LEN";
+
+    bool constant private TOKEN_TRANSFERABLE = false;
+    string constant private TOKEN_NAME = "Reputation Token";
+    string constant private TOKEN_SYMBOL = "REP";
+    uint256 constant private TOKEN_MAX_PER_ACCOUNT = uint256(-1);
+    uint64 constant private ONE_PCT = uint64(1e16);                         // 1%
+    uint64 constant private SUPPORT_REQUIRED = uint64(50 * ONE_PCT);        // 50%
+    uint64 constant private MIN_ACCEPTANCE_QUORUM = uint64(20 * ONE_PCT);   // 20%
```

Comment:
It seems a bit odd that you wouldn't be able to specify these parameters 🤷‍♂

Reply:
@lkngtn could you specify which parameters should be customizable by users?

Reply:
The voting parameters (support, quorum, duration) should all be user changeable, with the "default values" implemented in the Onboarding UI.